### PR TITLE
[merged] doc: Added feature overview.

### PR DESCRIPTION
**Pull request by ashcrow**

### CODE REVIEW

**Comment by ashcrow** [comment]
@devop-mmcgrath I'd like your thoughts on this too :smile:

File: doc/overview.rst [hunk 1]
```diff
@@ -7,13 +7,20 @@ Overview
...
+
+- Restart hosts in a container cluster
+- Upgrade hosts in a container cluster
+- Bootstrap new hosts into an existing container cluster
+- No agent required for hosts: All communication is done over SSH
+- Simple REST interface for automation
+- Service status for health checking
+- Plug-in based authentication framework
+- Command line interface for operators
+- Out of the box support for Atomic, RHEL, Fedora, and CentOS
```
Comment (mbarnes) [inline doc/overview.rst]: Maybe "Builtin support for..."?  Haven't bought Red Hat software in a box since the 90s.  :wink: 

List looks complete to me.

Comment (ashcrow) [inline doc/overview.rst]: Added in fixup.

**Comment by ashcrow** [comment]
:arrow_up:

**Comment by mbarnes** [comment]
LGTM  :+1:  Shall I wait to merge until @devop-mmcgrath has a look?

**Comment by ashcrow** [comment]
@mbarnes not required, but if he can take a gander before EOD that would be great.

**Comment by ashcrow** [comment]
@mbarnes if you are good with it I say go ahead and merge.

**Comment by mbarnes** [comment]
@cgwalters-bot r+

**Pull request closed without merging**